codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -143,26 +147,58 @@ var Worksheet = React.createClass({
             return false;
         }.bind(this));
     },
-    toggleEditMode: function(editMode) {
+    toggleEditMode: function(editMode, saveChanges) {
         if (editMode === undefined)
           editMode = !this.state.editMode;  // Toggle by default
 
+        if (saveChanges === undefined)
```

Comment:
nit: this can come in a future patch, but `typeof saveChanges === 'undefined'` seems to be better practice
http://stackoverflow.com/questions/27509/detecting-an-undefined-object-property